lib/dialects/mysql.ts

```diff
@@ -0,0 +1,123 @@
+import Knex from 'knex';
+import { SchemaInspector } from '../types/schema-inspector';
+import { MySQLTable } from '../types/table';
+import { MySQLColumn } from '../types/column';
+
+export default class MySQL implements SchemaInspector {
+  knex: Knex;
+
+  constructor(knex: Knex) {
+    this.knex = knex;
+  }
+
+  async tables() {
+    type RawTable = {
```

Comment:
why define type inside a method?

Reply:
Because this is only local to the current database dialect, and local to this specific method, as it matches the query that's being run